gapic-generator-java/scripts/BUILD.bazel

```diff
@@ -1,3 +0,0 @@
-package(default_visibility = ["//test/integration:__pkg__"])
```

Comment:
We may still need this folder for updating goldens. But as long as all the bazel tests are passing, we should be good.